extensions/binding/org.eclipse.smarthome.binding.sonos/README.md

```diff
@@ -88,6 +88,7 @@ The devices support the following channels:
 
 All supported Sonos devices are registered as an audio sink in the framework.
 Audio streams are treated as notifications, i.e. they are fed into the `notificationsound` channel and changing the volume of the audio sink will change the `notificationvolume`, not the `volume`.
+Note that the 'notificationsoundvolume' is set by the binding during each start of the system to equal the master volume (resulting in a barely audible level), making the use of a persistance policy obsolete. In order to control the 'notificationsoundvolume' users have to set it manually or by a rule.
```

Comment:
Looking at the above channels, you meant `notificationvolume` and not `notificationsoundvolume`, right?
Please also note the usage of the ` instead of ' for properly quoting the names.